pkg/util/azureutil/absfactory/client.go

```diff
@@ -18,6 +18,7 @@ import (
 	"fmt"
 
 	"github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/Azure/go-autorest/autorest/azure"
```

Comment:
Right now it looks like go-autorest is a transient dependency, this seems to make it a direct dependency.
Should this include an update to Gopkg.toml?

Reply:
Run `dep ensure` doesn't add that. Should I add it explicitly?

Reply:
No need to add go-autorest. it is vendored already via https://github.com/coreos/etcd-operator/blame/master/Gopkg.lock#L17